DigitalOffice/Services/UserService/Database/Entities/User.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace UserService.Database.Entities
+{
+    public class User
+    {
+        [Key]
+        public Guid Id { get; set; }
+        [Required]
+        public string Email { get; set; }
+        [Required]
+        public string FirstName { get; set; }
+        [Required]
+        public string LastName { get; set; }
+        public string MiddleName { get; set; }
+        public string Status { get; set; }
+        [Required]
+        public string PasswordHash { get; set; }
+        public Guid? AvatarFileId { get; set; }
+        [NotMapped]
+        public ICollection<UserCertificateFile> CertificatesFilesIds { get; set; }
+        [NotMapped]
+        public ICollection<UserPosition> PositionsIds { get; set; }
+        [NotMapped]
+        public ICollection<UserAchievement> AchievementsIds { get; set; }
```

Comment:
Why didn't you remove atribute NotMapped? If it stay in property, your columns will not be created.

Reply:
> tay in property, your columns will no

Do we want collection type columns?

Reply:
We use collection type, because use ICollection<T> for normal properties, just because its a well-known interface with minimal overheads, i.e. it is very slightly quicker to create an ICollection than a IList.